tests/functional/dev_setup.yml

```diff
@@ -6,16 +6,22 @@
     - vagrant_setup
   tasks:
 
-    - name: change ceph_stable to False
+    - name: change ceph_repository to 'dev'
       replace:
-        regexp: "ceph_stable:.*"
-        replace: "ceph_stable: False"
+        regexp: "ceph_repository:.*"
+        replace: "ceph_repository: dev"
         dest: "{{ change_dir }}/group_vars/all"
       when: change_dir is defined
 
-    - name: set ceph_dev to True
+    - name: set ceph_dev_branch to 'master'
```

Comment:
You don't need to set ``ceph_dev_branch`` or ``ceph_dev_sha1`` here. Those are passed into the playbook by using --extra-vars in tox.ini, which are populated from environment variables.